Earthfile

```diff
@@ -517,7 +517,7 @@ test-docker-compose:
     FROM earthly/dind:alpine
     COPY deploy/docker-compose.yml .
     WITH DOCKER --pull postgres \
-                --pull docker.redpanda.com/vectorized/redpanda:v22.3.11 \
+                --pull docker.redpanda.com/vectorized/redpanda:v23.1.13 \
```

Comment:
Perhaps it's better to use `WITH DOCKER --compose=deploy/docker-compose<-dev?>.yml --service=redpanda --service=postgres` and specify the versions once in the docker file? That way we don't have to specify the version multiple times.

Reply:
This is proving too much of a rabbit hole.  The `--compose` approach doesn't work because earthly doesn't like the `profiles:` flag in `docker-compose.yml`.  `RUN docker-compose` doesn't work because the default Ubuntu image doesn't contain docker compose.

Reply:
Ah damn, I encountered that a while ago. Their `alpine:dind` image has a later version of docker compose, and the version in their ubuntu images are literally one release behind for --profiles to work.

Go with your current approach for now.